packages/swapper/src/swappers/thorchain/getThorTradeQuote/getTradeQuote.test.ts

```diff
@@ -78,8 +78,13 @@ describe('getTradeQuote', () => {
 
     // Mock midgard api calls in 'getThorTxInfo' and 'getPriceRatio'
     mockedAxios.get.mockImplementation((url) => {
-      const isPoolResponse = url.includes('pools')
-      const data = isPoolResponse ? [ethMidgardPool, foxMidgardPool] : addressData
+      const isPoolsResponse = url.includes('pools')
+      const isPoolResponse = url.includes('pool') && !url.includes('pools')
+      const data = isPoolResponse
+        ? ethMidgardPool
+        : isPoolsResponse
+        ? [ethMidgardPool, foxMidgardPool]
+        : addressData
```

Comment:
Hard to grok. Replace with switch IIFE.

Reply:
Addressed in https://github.com/shapeshift/lib/pull/994/commits/d8032afa5b01359d403bbe6f4e9c5e3f90e3cf59